msbuild/Makefile

```diff
@@ -333,6 +333,7 @@ $(MAC_DESTDIR)$(MAC_FRAMEWORK_CURRENT_DIR)/lib/msbuild/%.dll: build/%.dll | $(MA
 all-local:: $(MSBUILD_PRODUCTS) test-xml
 
 .build-stamp: $(ALL_SOURCES)
+	cat ~/.config/NuGet/NuGet.Config
```

Comment:
The file might not exist on some systems, so probably better to do `|| true` so it doesn't fail the build. Sorry, should've mentioned this right away.